app/src/main/java/org/dhis2/usescases/searchTrackEntity/SearchTEActivity.java

```diff
@@ -259,24 +246,38 @@ protected void onResume() {
         } else {
             initSearchNeeded = true;
         }
+        if (teiMapManager != null) {
+            teiMapManager.onResume();
+        } else {
+            teiMapManager =  new TeiMapManager(
+                    new MapStyle(
+                            presenter.getProgram().style().color(),
+                            presenter.getTEIColor(),
+                            presenter.getSymbolIcon(),
+                            presenter.getEnrollmentColor(),
+                            presenter.getEnrollmentSymbolIcon()
+                    ));
+            teiMapManager.init(binding.mapView);
+            teiMapManager.setOnMapClickListener(this);
+        }
```

Comment:
@mmmateos couple of questions: 
- If we move it to onCreate, is this still needed?
- Can this be done inside the manager?